web-ng/src/index.html

```diff
@@ -25,5 +25,6 @@
 </head>
 <body>
   <app-root></app-root>
+  <script src="http://maps.googleapis.com/maps/api/js?key="></script>
```

Comment:
Is this basically used to import the APIs? Should/can this be placed inside the Map component?

Reply:
We spoke; `google` needs to be a global, so this needs to be defined at top level for now.